packages/components/src/utils/checkbox-radio-button-wrapper-utils.ts

```diff
@@ -0,0 +1,10 @@
+export const changeHandler = ({ target }: Event & { target: HTMLInputElement }): void =>
+  // workaround for Safari >= 15.5 -> attribute 'checked' is only set as property and not as attribute, this leads to rendering issues with keyboard navigation
+  document.querySelectorAll(`input[type=${target.type}][name=${target.name}]`).forEach((el: HTMLInputElement) => {
```

Comment:
Aren't double quotes still missing for the value of the `name` attribute ?
That's a regression for this bug https://github.com/porsche-design-system/porsche-design-system/issues/2390 which was fixed here https://github.com/porsche-design-system/porsche-design-system/pull/2443.